### PR TITLE
DBZ-1280 Lock timeout for SQL Server

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.jboss.org/browse/DBZ-1280

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@jpechane, LGTM! Only the doc upate missing, you know the drill :) Thanks!

**Comment by jpechane** [comment]
@gunnarmorling Added

File: documentation/modules/ROOT/pages/connectors/sqlserver.adoc [hunk 1]
```diff
@@ -1323,6 +1323,11 @@ Can be used to avoid snapshot interruptions when starting multiple connectors in
 |Specifies the maximum number of rows that should be read in one go from each table while taking a snapshot.
 The connector will read the table contents in multiple batches of this size. Defaults to 2000.
 
+|`snapshot.lock.timeout.ms`
+|`10000`
+|An integer value that specifies the maximum amount of time (in milliseconds) to wait to obtain table locks when performing a snapshot. If table locks cannot be acquired in this time interval, the snapshot will fail See link:#snapshots[snapshots]. +
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/sqlserver.adoc]:
```suggestion
|An integer value that specifies the maximum amount of time (in milliseconds) to wait to obtain table locks when performing a snapshot. If table locks cannot be acquired in this time interval, the snapshot will fail (also see link:#snapshots[snapshots]). +
```

**Comment by gunnarmorling** [comment]
Thanks, @jpechane!

**Pull request merged**